t/e2e/transactions/transactions_test.go

```diff
@@ -409,7 +410,8 @@ func (s *TransactionsTestSuite) TestSendEther() {
 }
 
 func (s *TransactionsTestSuite) TestSendEtherTxUpstream() {
-	if GetNetworkID() == params.StatusChainNetworkID {
+	// Only works on Ropsten.
+	if GetNetworkID() != params.RopstenNetworkID {
```

Comment:
So far, it has been run with Rinkeby network as well. I think this comment is invalid. And it should not be run in mainnet as it will certainly fail.

Reply:
Have to take a look at change history. Remember so far only adding the comment. Thanks.